apps/vc-api/package.json

```diff
@@ -22,7 +22,12 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "write-openapi": "ts-node ./scripts/write-open-api-json.ts && prettier -w docs/openapi.json",
-    "update-openapi": "npm run write-openapi; git add docs/openapi.json"
+    "update-openapi": "npm run write-openapi; git add docs/openapi.json",
+    "migration:generate:pg": "export DB_TYPE=POSTGRES; npm run migration:run && cd src/migrations/pg && typeorm-ts-node-commonjs -d ../../config/ormconfig.ts migration:generate -p",
```

Comment:
Same as above - ideally to configure by `.env` and have single `migration:generate`

Reply:
Yes, this was the first approach, but after exercising the creation of migrations a few times, I created two migration scripts in the `package.json` because when generating migrations, it is necessary to execute both one after another. Editing `.env` every time would be really annoying and prone to mistakes. :)

Reply:
> it is necessary to execute both for Postgres and SQLite

Could you give an example for such scenario?